CONTRIBUTING.md

```diff
@@ -245,13 +251,27 @@ If in doubt, you can always ask for guidance in the Pull Request or on
 Feel free to post a comment in the Pull Request to ping reviewers if you are
 awaiting an answer on something.
 
+Notes: if the reviewer mentions *nits* in their comments, that stands for
+"requests for small changes that are not essential". Usually these nits are
```

Comment:
I'm not sure the description is quite accurate. IMHO nits are just minor issues, whether the changes needed to address the minor issue are large or small is irrelevant. For example, comment typos for a PR that changes code would probably be considered a nit especially because comments don't affect the code, but it could be there are a lot of typos to fix or just one.